src/unistd.rs

```diff
@@ -214,6 +214,36 @@ pub fn write(fd: RawFd, buf: &[u8]) -> Result<usize> {
     Errno::result(res).map(|r| r as usize)
 }
 
+pub enum Whence {
+    SeekSet,
+    SeekCur,
+    SeekEnd
```

Comment:
Linux >= 3.1 also supports `SEEK_DATA` and `SEEK_HOLE` which might be worth supporting as well.